tools/meteor-services/config.js

```diff
@@ -8,179 +8,66 @@ var tropohouse = require('../packaging/tropohouse.js');
-//
-// We're not quite there yet though:
-// - When developing locally, you may need to set DISCOVERY_PORT (see
-//   getDiscoveryPort below)
-// - DEPLOY_HOSTNAME can still be set to override classic-style
-//   deploys
-// - The update/warehouse system hasn't been touched and still has its
-//   hardcoded URLs for now (update.meteor.com and
-//   warehouse.meteor.com). Really, it's debatable whether these
-//   should (necessarily) change when you change your universe name.
-
-var universe;
-var getUniverse = function () {
-  if (! universe) {
-    universe = "www.meteor.com";
-
-    if (files.inCheckout()) {
-      var p = files.pathJoin(files.getCurrentToolsDir(), 'universe');
-      if (files.exists(p)) {
-        universe = files.readFile(p, 'utf8').trim();
-      }
-    }
-  }
-
-  return universe;
-};
-
-var isLocalUniverse = function () {
-  return !! getUniverse().match(/^localhost(:([\d]+))?$/);
-};
-
-var localhostOffset = function (portOffset) {
-  var match = getUniverse().match(/^localhost(:([\d]+))?$/);
-  if (! match) {
-    throw new Error("not a local universe?");
-  }
-  return "localhost:" + (parseInt(match[2] || "80") + portOffset);
-};
-
-var getAuthServiceHost = function () {
-  if (! isLocalUniverse()) {
-    return universe;
-  } else {
-    // Special case for local development. Point
-    // $METEOR_CHECKOUT/universe at the place where you are running
-    // frontpage (eg, localhost:3000), and run the accounts server ten
-    // port numbers higher. Like so:
-    //   cd meteor-accounts
-    //   ROOT_URL=http://localhost:3010/auth curmeteor -p 3010
-    return localhostOffset(10);
-  }
-};
 
 // Given a hostname, add "http://" or "https://" as
 // appropriate. (localhost gets http; anything else is always https.)
-var addScheme = function (host) {
-  if (host.match(/^localhost(:\d+)?$/)) {
-    return "http://" + host;
+var addScheme = function (hostOrURL) {
+  if (hostOrURL.match(/^http/)) {
+    return hostOrURL;
+  } else if (hostOrURL.match(/^localhost(:\d+)?$/)) {
+    return "http://" + hostOrURL;
   } else {
-    return "https://" + host;
+    return "https://" + hostOrURL;
   }
 };
 
 var config = exports;
 _.extend(exports, {
-  // True if this the production universe (www.meteor.com)
-  isProduction: function () {
-    return getUniverse() === "www.meteor.com";
-  },
-
-  // The current universe name. Should be used for cosmetic purposes
-  // only (displaying to the user). If you want to programmatically
-  // derive configuration from it, add a new method to this file.
-  getUniverse: function () {
-    return getUniverse();
-  },
-
-  // Base URL for Meteor Accounts OAuth services, typically
-  // "https://www.meteor.com/oauth2". Endpoints include /authorize and
-  // /token.
+  // Base URL for Meteor Accounts OAuth services. Endpoints include /authorize
+  // and /token.
   getOauthUrl: function () {
-    return addScheme(getAuthServiceHost()) + "/oauth2";
+    return "https://www.meteor.com/oauth2";
```

Comment:
I dunno, I feel like over the past few years where these things have come from a series of functions I've had to sort through the file over and over trying to determine what the actual URLs are, and for clarity people ended up writing the URLs in comments anyway.  I agree with you in theory though, but not really in practice...